run/test/specs/message_requests_block.spec.ts

```diff
@@ -1,4 +1,5 @@
 import { androidIt, iosIt } from '../../types/sessionIt';
+import { BlockedContactsSettings, BlockUserConfirmation } from './locators';
 import { runOnlyOnAndroid, runOnlyOnIOS, sleepFor } from './utils';
 import { newUser } from './utils/create_account';
```

Comment:
```suggestion
import { sleepFor } from './utils';
```

Some unused functions

Reply:
Removed